### ISSUE TITLE
Missing app icons in kde

### ISSUE THREAD

**Issue by andia89** [numixproject/numix-core#2294]
Looking at [#580](https://github.com/numixproject/numix-icon-theme/issues/580) I recognized that a a few icons are actually app icons and not base icons. So I deleted them from there and consequentely open a new pull request with the missing icons here:
- [x] Charm
- [x] esd
- [x] kblogger
- [x] kbruch
- [x] kchart
- [x] kformula
- [x] kivio
- [x] kjournal
- [x] korgac
- [x] kplato
- [x] kpresenter
- [x] ksniffer
- [x] kspread
- [x] ktip
- [x] kverbos
- [x] kword
- [x] plasmagik
- [x] preferences-contact-list
- [x] preferences-desktop-cryptography
- [x] preferences-desktop-display-color
- [x] preferences-desktop-filetype-association
- [x] preferences-desktop-gaming
- [x] preferences-desktop-icons
- [x] preferences-desktop-mouse
- [x] preferences-desktop-printer
- [x] preferences-desktop-text-to-speech
- [x] preferences-plugin-script
- [x] preferences-system-cron
- [x] preferences-system-windows-move
- [x] preferences-web-browser-adblock
- [x] preferences-web-browser-cache
- [ ] preferences-web-browser-identification
- [x] preferences-web-browser-shortcuts
- [x] semn
- [x] strigi
- [x] utilities-desktop-extra

![unbenannt](https://cloud.githubusercontent.com/assets/6475757/8229404/b87056f0-15b6-11e5-8bc2-f5810b399a93.png)
##

**Comment by palob** [comment]
Maybe you can take off kchart, kformula, kivio, kplato, kpresenter, kspread and kwords. 

Those are applications from the former KOffice suite which is dead for 4 years now and succeeded by the Calliga Suite.
If needed nonetheless the icons could be done as symlinks to the Calligra icons.

**Comment by Foggalong** [comment]
Symlinks would be fine yeah. I wouldn't put it past someone out there to be still using the old office. We've had weirder things

**Comment by W-Floyd** [comment]
Checking through this list:
* `kbruch` has had its own icon for a while.
* `preferences-desktop-gaming` is already symlinked from `applications-games`

I'll prod at some of the others.

**Comment by W-Floyd** [comment]
`ksniffer` has not been updated in [~7~ 8 years](http://ksniffer.sourceforge.net/index.php?section=home). If anyone has ideas on a good symlink, that'd be great, but I'm coming up short on a generic icon for it.

**Comment by W-Floyd** [comment]
I'd like to get this resolved. Outstanding as of the moment are:  
- [x] ksniffer
- [x] preferences-desktop-filetype-association
- [x] preferences-plugin-script
- [x] preferences-system-cron
- [x]  preferences-web-browser-adblock
- [x]  preferences-web-browser-cache
- [ ]  preferences-web-browser-identification
- [x]  preferences-web-browser-shortcuts

`utilities-desktop-extra` may be checked, it was done as of #4032

**Comment by palob** [comment]
Haven't given KDE a shot in a long time. May do this again to see how we look on KDE nowadays.

**Comment by maxtron95** [comment]
@palob When my pull requests #5193, #5194, #5195, and #5196 gets merged, This issue should be complete.

**Comment by Foggalong** [comment]
Just waiting on adjustments to #5196 then this is good to go :+1:

**Comment by maxtron95** [comment]
The issue is complete

**Comment by palob** [comment]
So it's high time I take KDE for a spin again too see where the target has moved to :-)
